2025/README.md

```diff
@@ -0,0 +1,16 @@
+**Interop 2025 has not yet launched and this documentation is still being finalized. Please do not submit proposals yet.**
+
+# Interop 2025
+
+Interop 2025 is an effort to increase interoperability across browsers in key technical areas that are of high priority to web developers and end users. More details on the project is available in the [Interop Project README](https://github.com/web-platform-tests/interop/blob/main/README.md).
+
+## Timeline
+
+Interop 2025 will open for proposals on September 12, 2024 and the overall timeline is as below. Please note that the timelines are subject to change.
+- Proposal submission window (3 weeks): Sept 12th, 2024 to Oct 3rd, 2024
```

Comment:
Was the intent that proposals can still be submitted on Oct 3rd? If so maybe this could say Sept 12th, 2024 through Oct 3rd, 2024.

Reply:
If we want to exclude the 3rd, it's perhaps clearer to say Sept 12th, 2024 through Oct 2nd, 2024.

Reply:
Updated